.gitignore

```diff
@@ -1,6 +1,7 @@
 *.egg-info
 *.pyc
-/.pytest_cache
 /.coverage
+/.idea/
```

Comment:
this is currently generated from a (private, maybe one day public) template.  let's leave this file alone  (same in your other PR too! thanks!)

it'll get undone when I finish my "updates to the template updates all the repos" project (that's getting surprisingly close)

Reply:
Sure